bin/npg_deletable_dr_runs

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use English qw(-no_match_vars);
+use Getopt::Long;
+use Pod::Usage;
+
+use npg_tracking::Schema;
+
+our $VERSION = '0';
+
+my $help;
+my $verbose;
+GetOptions ('verbose' => \$verbose,
+            'help|?'   => \$help)
+  or croak("Error in command line arguments\n");
+
+pod2usage(1) if $help;
+
+#my $rs=npg_tracking::Schema->connect()->resultset("Run")->search();
+my $rs=npg_tracking::Schema->connect()->resultset(q(RunStatusDict))
+    ->search({description=>[q[run cancelled],q[data discarded],q[qc complete]]})
+    ->related_resultset(q(run_statuses))->search({q(run_statuses.iscurrent)=>1})
+    ->related_resultset("run")->search({q(run.folder_path_glob)=>{LIKE=>q(%/esa-%)}});
+my $total = $rs->count;
+my $run;
+my $found = 0;
+while (my $run=$rs->next()) {
+    my $status = $run->current_run_status_description || "";
+    my $tag = $run->is_tag_set("staging");
+    my $run_folder = $run->folder_name || "";
+    my $path_glob = $run->folder_path_glob || "";
+    if ($run_folder and !$tag ) {
+        print $run_folder, "\t", $run->id_run, "\t", $run->folder_path_glob, "\t", $status, "\n";
+        $found++;
+    }
+}
+
+if ($verbose) { print STDERR "\nFound ", $found, " out of ", $total, " records\n"; }
```

Comment:
Probably the verbose option is not needed, the found number follows from the number of lines, the total is not telling anything important. If left should be added to a list of options in POD